index.html

```diff
@@ -13,7 +13,7 @@
         <img src="icons/Icon - Menu.svg" alt="" />
       </nav>
       <section class="container-intro">
-        <h1>Hey there.<br />I’m Victor.<br /></h1>
+        <h1>Hey there.<br/>I’m Victor.<br/></h1>
         <p>
```

Comment:
- [ ] Kindly ensure that when next you make changes to a repo, all the changes are done on a new separate branch, independent of the `main` branch,  this will ensure those changes appear under the `Files changed` tab and this will enable the reviewer to point out reviews, suggestions, and feedback under the `Files-changed` tab, enabling you and the reviewer have a hitch-free review session. 😃 

Summarily: do not hand code directly into your `main` branch... Only introduce code to it (`main`) by merging through a pull request.

This will also make sure you're using and following the correct GitFlow.